ipatests/test_integration/test_trust.py

```diff
@@ -106,6 +107,36 @@ def check_range_properties(self, realm, expected_type, expected_size):
         expected_text = 'iparangetype: %s\n' % expected_type
         assert expected_text in result.stdout_text
 
+    def mod_idrange_auto_private_group(
+        self, auto_priv_grp_val='false'
+    ):
+        """
+        Set the auto-private-group option of the default trusted
+        AD domain range and restart sssd
+        """
+        rangename = self.existing_rangename
+        valid_msg = "Auto private groups: "
+        error_msg = "ipa: ERROR: no modifications to be performed"
+        cmd = ["ipa", "idrange-mod", rangename,
+               "--auto-private-groups", auto_priv_grp_val]
+        result = self.master.run_command(cmd, raiseonerr=False)
+        try:
+            self.master.run_command(cmd)
+            self.master.run_command(["systemctl", "restart", "sssd"])
+        except CalledProcessError as e:
+            # ignore error if the auto-private-group is already set
+            # to the expected value
+        if error_msg not in e.stderr:
```

Comment:
This line and the following need to have +4chars indentation in order to be executed in the except clause.